spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/LinkNode.java

```diff
@@ -50,7 +50,7 @@ public Type getType() {
 	}
 
 	public enum Type {
-		input, output, error, discard
+		input, output, error, discard, route
```

Comment:
I was thinking that we might want to render these links differently - maybe with dotted lines or a different color - to distinguish between the default output channel and the router channels.

Yes, I will add code for the RLR.